packages/forma-36-react-components/src/components/DateTime/DateTime.test.tsx

```diff
@@ -5,6 +5,22 @@ import { DateTime } from './DateTime';
 
 const exampleDate = new Date('2020-04-09T16:17:18.912Z');
 
+const monkeyPatch = (errorMessage: string) => {
+  // Throw expected error silently
+  const originalFunction = console.error;
+  const spy = jest.spyOn(console, 'error');
+  spy.mockImplementation((error) => {
+    if (error.toString().includes(errorMessage)) {
```

Comment:
```suggestion
    if (error.message === errorMessage) {
```

Do you think it will be useful to be able to check for a substring?

Reply:
I like that it allows more flexibility so I don't have to look out for a missing empty space or some punctuation marks. I don't see any risk in there. If someone calls `monkeyPatch('error')` it's quite obvious that this is not specific enough. On the other hand, someone is able to call monkeyPatch('') to quickly turn off all errors.